ts/nni_manager/common/trainingService.ts

```diff
@@ -8,7 +8,7 @@
  */
 type TrialJobStatus = 'UNKNOWN' | 'WAITING' | 'RUNNING' | 'SUCCEEDED' | 'FAILED' | 'USER_CANCELED' | 'SYS_CANCELED' | 'EARLY_STOPPED';
 
-type LogType = 'TRIAL_LOG' | 'TRIAL_STDOUT' | 'TRIAL_ERROR';
+type LogType = 'TRIAL_LOG' | 'TRIAL_STDOUT' | 'TRIAL_ERROR' | 'MODEL.onnx';
```

Comment:
We can remove it if the parameter type is 'string' or keep this type for parameter and rename it as "FileName"?

Reply:
Just removed LogType as I think it's no longer used.